docs/about/contributing.md

```diff
@@ -47,6 +47,14 @@ pip install --editable .
 This will install MkDocs in development mode which binds the `mkdocs` command
 to the git repository.
 
+If _mkdocs-redirects_ is not already present run the following command (**strongly** recommended that you do
+this within a [virtualenv]).
+
+```bash
+pip install mkdocs-redirects
```

Comment:
This part of the change is invalid.

This is just `pip install -r requirements/project.txt` which is already presumed

https://github.com/mkdocs/mkdocs/blob/f8b9e967186392d0013a2ca1961a5113e1baf602/requirements/project.txt#L11